package.json

```diff
@@ -136,13 +136,13 @@
     "dependencies": {
         "@0x/api-utils": "^0.0.2",
         "@0x/assert": "^3.0.34",
-        "@0x/asset-swapper": "^16.60.1",
-        "@0x/contract-addresses": "^6.14.0",
-        "@0x/contract-wrappers": "^13.20.2",
```

Comment:
nitpick: we update minor versions for AS, addresses, and 0x-contracts but patch for wrappers.

Reply:
looks like contract-wrappers change was just dependencies upgrade:

https://github.com/0xProject/protocol/blob/development/packages/contract-wrappers/CHANGELOG.json